src/components/layout/header.tsx

```diff
@@ -1,37 +1,57 @@
 import { Link, NavLink } from 'next-theme-ui';
 import { useRouter } from 'next/router';
-import { Container, Flex } from 'theme-ui';
+import { Box, Container, Flex } from 'theme-ui';
+import { Booking } from '../booking';
 import { Logo } from '../logo';
 
 export const Header: React.FC = () => {
   const { pathname } = useRouter();
+  const color = pathname === '/' ? 'green100' : 'green20';
   return (
-    <Container variant="readable">
-      <Flex
-        sx={{ pt: 2, justifyContent: 'space-between', alignItems: 'center' }}
-      >
-        <Link href={'/'}>
-          <Logo
+    <Box
+      sx={{
+        pt: 7,
+        pb: 2,
+        backgroundColor: color,
+      }}
+    >
+      <Container variant="header">
+        <Flex sx={{ alignItems: 'center', gap: 5 }}>
+          <Link href={'/'}>
+            <Logo
+              sx={{
+                maxWidth: '100px',
+                cursor: 'pointer',
+                height: '40.87px',
+                color: 'text',
+              }}
+            />
+          </Link>
+          <NavLink
+            href={'/about'}
             sx={{
-              maxWidth: '100px',
-              cursor: 'pointer',
-              height: '40px',
-              color: 'text',
+              color: pathname.includes('about') ? '#2A3BCC' : 'text',
```

Comment:
It's probably better to add `#2A3BCC` to the theme?

Reply:
It was added to the theme as 'blue100', but missed this in the sweep of hexcodes.